### PR TITLE
ICC: sync CMS Signatures with latest CMM signature registry

### PR DESCRIPTION
**Pull request by armijnhemel**
http://www.color.org/registry/signature/TagRegistry-2021-03.pdf

### CODE REVIEW

**Comment by KOLANICH** [comment]
You may want to reuse something from https://github.com/KOLANICH/kaitai_struct_formats/blob/broadcom_patchram/hardware/bluetooth/updateVendorIds.py

**Comment by armijnhemel** [comment]
> You may want to reuse something from https://github.com/KOLANICH/kaitai_struct_formats/blob/broadcom_patchram/hardware/bluetooth/updateVendorIds.py
> 
> and especially from https://github.com/KOLANICH-tools/MAPITagsScraper.py/tree/master/MAPITagsScraper/sources

I specifically only looked at the official CMS Signatures document (from March 2021), as the ICC database contains many many entries and I didn't have the time to add all of them.

**Comment by KOLANICH** [comment]
By the links I have sent are not the documents, these are scripts scraping and generating enums for other specs. I feel like it may make sense to create an analogue of them for this spec too.

File: image/icc_4.ksy [hunk 1]
```diff
@@ -97,14 +97,18 @@ types:
         0x44676f53: logosync_cmm #LgoS
         0x48444d20: heidelberg_cmm #HDM
         0x6C636d73: little_cms_cmm #lcms
+        0x52494d58: reficcmax_cmm #RIMX
+        0x44494d58: demoiccmax_cmm #DIMX
         0x4b434d53: kodak_cmm #kcms
```
Comment (generalmimon) [inline image/icc_4.ksy:102]: BTW, this is apparently uppercase:

```suggestion
        0x4b434d53: kodak_cmm #KCMS
```

Comment (armijnhemel) [reply image/icc_4.ksy]: Fixed

**Review by generalmimon — APPROVED** [review]
@armijnhemel LGTM, thanks for the contribution!

**Pull request merged**